crates/store/re_chunk_store/src/query.rs

```diff
@@ -430,8 +430,6 @@ impl ChunkStore {
         entity_path: &EntityPath,
         component_name: ComponentName,
     ) -> Vec<Arc<Chunk>> {
-        re_tracing::profile_function!(format!("{query:?}"));
```

Comment:
If the profiling has too much overhead on these I suggest adding a comment warning off other from re-adding them again